ddtrace/internal/telemetry/writer.py

```diff
@@ -0,0 +1,221 @@
+import logging
+from typing import ClassVar
+from typing import Dict
+from typing import List
+from typing import Optional
+
+from ddtrace.internal import forksafe
+
+from ..agent import get_connection
+from ..agent import get_trace_url
+from ..compat import get_connection_response
+from ..compat import httplib
+from ..encoding import JSONEncoderV2
+from ..logger import get_logger
+from ..periodic import PeriodicService
+from ..utils.formats import get_env
+from ..utils.time import StopWatch
+from .request import create_telemetry_request
+from .request import get_headers
+
+
+log = get_logger(__name__)
+
+
+def _get_interval_or_default():
+    return float(get_env("instrumentation_telemetry", "interval", default=60))
```

Comment:
```suggestion
    return float(get_env("instrumentation_telemetry", "interval_seconds", default=60))
```

Also can we just explicitly use an env variable rather than `get_env`